src/components/ActionButtonBar.vue

```diff
@@ -103,6 +125,25 @@ export default class ActionButtonBar extends Vue {
       this.$emit("fileLoaded", loadedFile);
     };
 
+    //iterates through file until it finds the . which indicates start of file type
```

Comment:
What if there are multiple dots in the filename? You need to find the last one...

file.name.split('.').pop(); // seems to be a common method